packages/http-client/src/http-client.ts

```diff
@@ -121,6 +122,32 @@ export function execute<ReturnT>(executeFn: ExecuteHttpRequestFn<ReturnT>) {
   };
 }
 
+function encodeRequestQueryParameters(
+  parameter: Record<string, string> | undefined
+): Record<string, string> | undefined {
+  if (parameter) {
+    return Object.keys(parameter).reduce(
+      (encodedParams, key) => ({
+        ...encodedParams,
+        [key]: encodeURIComponent(parameter[key])
+      }),
+      {}
+    );
```

Comment:
```suggestion
    return Object.fromEntries(Object.entries(parameter).map(
      ([key, value]) => 
        [encodeURIComponent(key), encodeURIComponent(parameter[key])]
      })
    ));
```
I like `Object.entries` better and I think `.fromEntries` and `.map` are nicer than `.reduce`.

Reply:
I agrre the `fromEntries` is more clear then a reduce.